next-packages/cache/src/rule-sheet.js

```diff
@@ -22,30 +22,56 @@ let ruleSheet: StylisPlugin = (
   line,
   column,
   length,
-  at,
-  depth
+  ns,
+  depth,
+  at
 ) => {
   switch (context) {
     case -1: {
       current = []
       break
     }
-    case 2:
-      if (at === 0) return content + delimiter
+    // property
+    case 1: {
+      switch (content.charCodeAt(0)) {
+        case 64: {
+          // @import
+          if (depth === 0) {
+            current.push(content + ';')
+            return ''
+          }
+          break
+        }
+        // charcode for l
+        case 108: {
```

Comment:
this https://github.com/xtuc/charcodes/tree/master/packages could be used to better distinguish char codes and plugin phases/contexts here (both being numbers)

Reply:
Not sure if it’s worth it, especially because we would have to add it as a regular dependency so that people who use flow wouldn’t get flow errors